13output.tex

```diff
@@ -464,7 +464,7 @@
 	\begin{itemize}
 		\item This section shows the time series of mean length-at-age for each platoon. At the bottom is the average mean size as the weighted average across all platoons for each sex.
 	\end{itemize}
-	\item Age Length Key
+	\item Age-length-key (\gls{alk})
```

Comment:
I am not sure why you write it out and have the gls entry. You can force the gls entry to be the full word if you want or the full word plus the abbreviation but this will write age-length-key twice if it is the first instance.